images/oc-build-deploy-dind/openshift-templates/cli-persistent/deployment.yml

```diff
@@ -135,6 +135,12 @@ objects:
                   memory: 10Mi
                 limits:
                   memory: 8Gi
+              readinessProbe:
+                initialDelaySeconds: 15
```

Comment:
let's use much less than 15secs here, having 15 secs here means k8s will wait for 15sec after the container has started before it tries the first time if the container is ready, aka this delays everything by 15 secs.
I think 1 sec is perfectly fine here.